d3ploy/no_inst.py

```diff
@@ -247,13 +243,11 @@ def calc_diff(self, commod, time):
         demand : double
             The calculated demand of the demand commodity at [time]
         """
-        # what is this?
         if time not in self.commodity_demand[commod]:
             t = 0
             self.commodity_demand[commod][time] = eval(self.demand_eq)
         if time not in self.commodity_supply[commod]:
-            t = 0
-            self.commodity_supply[commod][time] = eval(self.demand_eq)
+            self.commodity_supply[commod][time] = 0
```

Comment:
This can result in a situation in which the initial demand is 0. Why was this changed?

Reply:
isn't it just changing the supply? I think the problem was that it would match the demand value with the supply (since `self.commodity_supply[commod][time]` and `self.commodity_demand[commod][time]` was set to the same value, `eval(self.demand_eq)` at time t=0).  Can you explain how this causes the initial demand to be zero?

Reply:
Oh, right. I'm pretty sure that supply line is there because we can't operate on the decision phase just yet. It basically just ensures that there is at least something there if all of the facilities post their production after the no_inst runs it's tock. 

Because otherwise I don't think there is a reason for this section of code at all.